code/ui/blocks/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "module": "esnext",
```

Comment:
@ndelangen I changed this so TS wouldn't complain that I was referencing `import.meta.env`, do you know if changing this will break something somewhere else?

Reply:
is completely ignored 👍